docs/sources/mimir/migration-guide/migrating-from-cortex.md

```diff
@@ -228,76 +232,32 @@ You can update to the Grafana Mimir Helm chart from the Cortex Helm chart.
    b. Use `mimirtool` to update the configuration.
 
    ```bash
-   mimirtool config convert --yaml-file cortex.yaml
+   mimirtool config convert --gem --yaml-file cortex.yaml --yaml-out mimir.yaml
    ```
 
-   c. Place the updated configuration under the `mimir.config` key at the top level of your Helm values file.
-
-   > **Note:** The Grafana Mimir Helm chart expects the configuration as a string value.
-   > You can provide a literal block string with the `|` symbol.
+   c. Clean up generated yaml config
+   We have to remove some fields generated by `mimirtool config convert` for helm chart installation, since
+   mimir-distributed helm chart has already set the default value for them. Use the following script to clean up
+   those fields.
```

Comment:
```suggestion
   You have to remove some fields that are generated by `mimirtool config convert` for the Helm chart installation, because the
   `mimir-distributed` Helm chart has already set the default value for them. Use the following script to clean up
   those fields:
```